### PR TITLE
Integrate MAY-2022 ASB patches

### PR DESCRIPTION
**Pull request by rrsenapati1**
Tracked-On: OAM-101831
Signed-off-by: Senapati, Rasmiranjan <rasmiranjan.senapati@intel.com>

### CODE REVIEW

File: aosp_diff/preliminary/packages/providers/MediaProvider/03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch [hunk 1]
```diff
@@ -0,0 +1,120 @@
+From 15dc9f3496376832f8a1fe2f15d0f66c57248f0c Mon Sep 17 00:00:00 2001
+From: Ivan Chiang <chiangi@google.com>
+Date: Mon, 16 Aug 2021 15:13:32 +0800
+Subject: [PATCH] Handle content uris without id in checkUriPermission
+
```
Comment (SaliniVenate) [inline aosp_diff/preliminary/packages/providers/MediaProvider/03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch:5]: This is seen as a already applied patch.
Project ./packages/providers/MediaProvider
        Already applied         01_0001-Handle-content-uris-without-id-in-checkUriPermission.patch
        Applying                02_0002-Restrict-legacy-apps-to-insert-files-to-other-privat.patch
        Already applied         03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch

File: aosp_diff/preliminary/system/sepolicy/11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch [hunk 2]
```diff
@@ -0,0 +1,50 @@
+From b2fd7a5acc84b1b21563ccc893a97bb93b33d64c Mon Sep 17 00:00:00 2001
+From: Gregory Montoir <gmontoir@google.com>
+Date: Mon, 6 Sep 2021 09:25:13 +0800
+Subject: [PATCH] Allow adbd to access /proc/net/{tcp,tcp6,udp,udp6}
+
+File accesses go through com.android.ddmlib.SyncService for CTS
+ListeningPortsTest.
```
Comment (SaliniVenate) [inline aosp_diff/preliminary/system/sepolicy/11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch:7]: This is also already applied.
Project ./system/sepolicy
 Already applied         11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch

**Review by SaliniVenate — CHANGES_REQUESTED** [review]

**Comment by SaliniVenate** [comment]
@rrsenapati1 , recently an AOSP rebase was done on celadon-S and because of that many patches (apart from 2 commented above) are seen as already applied. Below is the list. Please clean up those patches also.
        Already applied         01_0001-Move-slice-increments-after-completing-header-parsin.patch
        Already applied         06_0006-Better-buffer-overrun-prevention.patch
        Already applied         07_0007-Safetynet-logging-for-b-204445255.patch
        Already applied         08_0008-Prevent-out-of-bounds-read.patch
        Already applied         05_0005-Add-missing-size-check-when-parsing-staged-aliases.patch
        Already applied         06_0006-Controls-Do-not-recreate-intent.patch
        Already applied         01_0001-Fix-Managed-Provisioning-crash-on-OTA.patch
        Already applied         01_0001-Rephrase-dialog-message-of-clear-storage-dialog-for-.patch
        Already applied         02_0002-Add-non-system-overlay-flag-to-InstallCaCertificateW.patch
        Already applied         03_0003-Import-translations.-DO-NOT-MERGE-ANYWHERE.patch
        Already applied         01_0001-Handle-content-uris-without-id-in-checkUriPermission.patch
        Already applied         03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch
        Already applied         03_0003-security-Use-After-Free-in-btm_sec_-dis-connected.patch
        Already applied         04_0004-Reset-the-IRK-after-all-devices-are-unpaired.patch
        Already applied         01_0001-Tag-mismatch-in-NFA_SendRawFrame.patch
        Already applied         11_0011-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch
        Already applied         01_0001-Revert-Revert-Add-automatic-default-value-for-char-t.patch
        Already applied         02_0002-Fix-sc-dev-plus-aosp-builds.patch
        Already applied         0002-Set-the-primary-display-as-internal-display-for-AOSP.patch

**Review by SaliniVenate — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
Integrating Google security patches.
LGTM

**Pull request merged**